### DISCUSSION TITLE
I setup Inbucket on my dedicated vm, now I need get ride of junk what I receive

### DISCUSSION THREAD

**Discussion by kashcode** [inbucket/inbucket#349] [Q&A] [answered]
Hi
![image](https://user-images.githubusercontent.com/7689209/221162465-936ef28a-9ad7-49db-a680-efe6ec114012.png)
Marked with red is not my email I somehow received.
What configuration I need made to get ride of this kin of messages?

**Comment by jhillyerd** [comment, marked as answer]
By default, Inbucket accepts mail to and from any address, which makes it look like an [open relay](https://www.techopedia.com/definition/1699/open-relay) -- this attracts a lot of spam, as spammers think they can relay spam to other mail hosts through your server.

Inbucket is a testing tool, it isn't really designed to be exposed to the open internet, so it doesn't have any spam filtering features.   

You can improve things somewhat by only accepting mail for the domain your are hosting, by setting env vars:

INBUCKET_SMTP_DEFAULTACCEPT=false
INBUCKET_SMTP_ACCEPTDOMAINS=example.com
